.github/workflows/tests.yml

```diff
@@ -0,0 +1,31 @@
+name: Tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.7', '3.8']
```

Comment:
Can we also add 3.6 here?

Reply:
Tja. I believe we need it here because the GitHub Actions environment actually **installs** the designated Python environments, which `nox` doesn't do. So, I've just added 3.6 here.